src/current/v22.2/changefeed-messages.md

```diff
@@ -134,26 +136,61 @@ Note that the original CockroachDB column definition is also included in the sch
 
 ### Schema changes with column backfill
 
-When schema changes with column backfill (e.g., adding a column with a default, adding a computed column, adding a `NOT NULL` column, dropping a column) are made to watched rows, the changefeed will emit some duplicates during the backfill. When it finishes, CockroachDB outputs all watched rows using the new schema. When using Avro, rows that have been backfilled by a schema change are always re-emitted.
+When schema changes with column backfill (e.g., adding a column with a default, adding a computed column, adding a `NOT NULL` column, dropping a column) are made to watched rows, CockroachDB emits a copy of the table using the new schema. When using Avro, rows that have been backfilled by a schema change are always re-emitted.
+
+{{site.data.alerts.callout_info}}
+Schema changes that do **not** use the declarative schema changer by default will trigger a changefeed to emit a copy of the table being altered as well as a copy of the table using the new schema. For a list of supported schema changes, refer to the [Declarative schema changer](online-schema-changes.html#declarative-schema-changer) section.
```

Comment:
Is "by default" important/correct here? Is there a case where you can force the declarative schema changer to be used but changefeeds still emit duplicates as though you hadn't?

Reply:
So, yes, I believe, so. According to the schema changer docs there are cluster settings to disable/enable the declarative schema changer, but by default the list on that page will use the dsc